azure-pipelines.yml

```diff
@@ -7,11 +7,11 @@ schedules:
     displayName: Nightly build
     branches:
       include:
-        - master
+        - branch-5.15
     always: true
 
 trigger:
-  - master
+  - branch-5.15
```

Comment:
The branch configuration is now merged to master. Can you try to push empty commit to a `branch-test` branch to see if it gets build without PR?